pkg/relayer/session/sessiontree.go

```diff
@@ -273,8 +273,10 @@ func (st *sessionTree) Delete() error {
 	// This was intentionally removed to lower the IO load.
 	// When the database is closed, it is deleted it from disk right away.
 
-	if err := st.treeStore.Stop(); err != nil {
-		return err
+	if st.treeStore != nil {
```

Comment:
Can we add an error log if `treeStore == nil`? 
I think it would be helpful if this condition is reached for whatever reason in the future.

Reply:
+1